openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/exonerations.py

```diff
@@ -30,6 +30,21 @@
 
 
 @reference_formula
+class exoneration_cotisations_patronales_geographiques(SimpleFormulaColumn):
+    column = FloatCol
+    entity_class = Individus
+    label = u"Exonérations de cotisations patronales dépendant d'une zone géographique"
+    url = "https://www.apce.com/pid815/aides-au-recrutement.html?espace=1&tp=1"
+
+    def function(self, simulation, period):
+
+        return period, (
+            exoneration_cotisations_patronales_zfu + exoneration_cotisations_patronales_zrd +
```

Comment:
@benjello I reject the pull request, sorry. The tests fail here, because you did not call calculate on these variables.